Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN cd /root && \
     unzip vault_${vault}_linux_${goplatform}.zip && \
     mv vault /usr/local/bin/ && \
     go install github.com/go-delve/delve/cmd/dlv@latest && \
+    go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest && \
```

Comment:
Nice that the same image can be used for linting.

There is also a golangci-lint github action available though. See for example https://github.com/lightningequipment/circuitbreaker/blob/master/.github/workflows/golangci-lint.yml

Reply:
I've reworked this using a github action, and rebase on master so that I can base #11 on top of this PR instead.